keyboards/copenhagen_click/click_pad_v1/config.h

```diff
@@ -18,6 +18,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 #include "config_common.h"
+#include "pro_micro.h"
```

Comment:
Could these changes be pulled out and put into a different PR?

Reply:
Hi drashna, thank you for reviewing my PR. I actually didn't intend to edit any other boards other than mine, but somehow some merge artifact got into my project, and I'm pretty confused on how to resolve it. Should i checkout this board just like how you show it below? Thank you, I'm sorry for this inconvenience.

About the custom matrix, yes, how the matrix is wired up is the reason. 

Im using a multiplexer on my matrix. It was inspired by Meira and Ronin, but i wired it slightly different. I use Col2Row configuration and connected all the columns to the multiplexer. So instead of digitally switching between column to read them, I use the multiplexer to analogously switch between columns.

Reply:
Okay, not a problem, stuff happens! 

For this, the simplest mething should be to just run:
```
git checkout upstream/master keyboards/copenhagen_click/click_pad_v1/config.h
```

That should pull the correct from from upstream (which should hopefully be set, if not, then `git remote add upstream https://github.com/qmk/qmk_firmware.git` followed by a `git fetch -all` should fix that)

Reply:
Thank you for your help! I've checked out the merge artifacts in problem. I hope everything is alright now.